include/verilogAST.hpp

```diff
@@ -645,6 +645,19 @@ class ModuleInstantiation : public StructuralStatement {
   ~ModuleInstantiation(){};
 };
 
+class IfDef : public StructuralStatement {
+  public:
+    std::string var;
```

Comment:
maybe rename `var` to `condition_string` or `cond` to make it more generic, and clear that it's just a serialized string, not necessarily a var (e.g. i imagine that currently, if we wanted to do logic, we'd just pass it as a string e.g. `"x | y"`).

Reply:
oops just noticed this is a vector. my bad.